.github/workflows/build-on-push.yml

```diff
@@ -0,0 +1,64 @@
+# Workflow to automatically create deliverables
+name: Build on push
+
+on:
+  [push, pull_request]
+
+jobs:
+  build:
+    name: Assembling artifacts
+    runs-on: ubuntu-20.04
+
+    # Note, to satisfy the asset library we need to make sure our zip files have a root folder
+    # this is why we checkout into demo/godot-xr-tools
+    # and build plugin/godot-xr-tools
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          path: demo/godot-xr-tools
+      - name: Create XR tools plugin
+        run: |
+          mkdir plugin
+          mkdir plugin/godot-xr-tools
+          mkdir plugin/godot-xr-tools/addons
+          cp -r demo/godot-xr-tools/addons/godot-xr-tools plugin/godot-xr-tools/addons
+          cp demo/godot-xr-tools/LICENSE plugin/godot-xr-tools
+          cp demo/godot-xr-tools/CONTRIBUTORS.md plugin/godot-xr-tools
+          cp demo/godot-xr-tools/VERSIONS.md plugin/godot-xr-tools
+          rm -rf demo/godot-xr-tools/.git
+          rm -rf demo/godot-xr-tools/.github
+      - name: Add OpenXR Plugin
+        run: |
+          wget --header "Authorization: token ${{ secrets.GITHUB_TOKEN }}" "https://github.com/GodotVR/godot_openxr/releases/download/1.3.0/godot-openxr.zip"
```

Comment:
For now we'll need to hardcode the version of the OpenXR plugin we embed in our demo asset.

Reply:
I think this works fine for now. I was thinking it could be a build-job variable accessed by the yaml; however I can't find how to set variables through the web; and even if it worked, we might want to have different versions specified for different branches - something embedding in the yaml is capable of achieving.

It does mean we have to update the yaml when the repo wants to jump forwards a rev; but that seems a small price to pay for having the explicit control in branches.